File: examples/pascal/constructs/scope_shadowing.pas
// Ported from examples/lumen/constructs/scope_shadowing.lm by scripts/port_examples.py; edit the Lumen original, not this file.
var x: integer;
begin
    writeln(1);
    x := 10;
    writeln(x);
    if true then begin
        x := 20;
        writeln(x);
    end;
    writeln(x);
end.

File: examples/php/constructs/scope_shadowing.php
<?php
// Ported from examples/lumen/constructs/scope_shadowing.lm by scripts/port_examples.py; edit the Lumen original, not this file.
print(1 . "\n");
$x = 10;
print($x . "\n");
if (true) {
    $x = 20;
    print($x . "\n");
}
print($x . "\n");
